docs/source/Internals.md

```diff
@@ -111,6 +111,13 @@ is 0, that means the field is not present in this object, and the
 default value is return. Otherwise, the entry is used as offset to the
 field to be read.
 
+### Unions
+
+FlatBuffers assigns a integer identifier to each of the choices in the union.
```

Comment:
A union doesn't really have its own encoding, since it is implemented as two fields, an enum, and the actual element (always an offset). It is thus defined by how those 2 are encoded. And you can't say its pre-fixed by an 8-bit index, since that is subject to field alignment and field ordering (they don't need to be adjacent) in the wire format. So not sure it needs this section.

Reply:
@aardappel Thanks for the clarifications. I tried to re-phrase the paragraph to make it more accurate. It still believe it would be nice to mention how this is encoded as i.e. what you just said is not clear from the current documentation.